src/argilla/client/feedback/dataset.py

```diff
@@ -782,7 +785,9 @@ def format_as(self, format: Literal["datasets"]) -> "Dataset":
         raise ValueError(f"Unsupported format '{format}'.")
 
     @requires_version("huggingface_hub")
-    def push_to_huggingface(self, repo_id: str, generate_card: Optional[bool] = True, *args, **kwargs) -> None:
+    def push_to_huggingface(
+        self, repo_id: str, generate_card: Optional[bool] = True, *args, **kwargs
+    ) -> Union[str, None]:
```

Comment:
And here there's no need to return `Union[str, None]`, but `str`, as previously we were not returning `None` as is, just not returning anything.

```suggestion
    ) -> str:
```